### ISSUE TITLE
I'm not sure if the training loss is right and the process of training is running as expected.

### ISSUE THREAD

**Issue by linjinjin123** [haowen-xu/donut#5]
![image](https://user-images.githubusercontent.com/10062840/44652534-b22f8680-aa1e-11e8-9645-6a708f00781f.png)
I am using a demo dataset to train a Donut, but got a negative training loss, but in the paper the training loss seems to be positive.
![image](https://user-images.githubusercontent.com/10062840/44652544-b6f43a80-aa1e-11e8-9000-c619aa0eaa81.png)
I wonder if a negative value of the training loss  is right? Is there anything wrong?
